cpo/lib/cloud_pak_for_data/cpd_4_0_0/cpd_manager.py

```diff
@@ -717,7 +709,70 @@ def _create_custom_resource(self, project: str, custom_resource: CustomResource)
         logger.info(f"Creating custom resource {custom_resource.kind} '{name}'")
         logger.debug(f"Sending JSON object:\n{body_str}")
 
-        self._openshift_manager.create_custom_resource(project, custom_resource)
+        self._openshift_manager.create_namespaced_custom_resource(project, custom_resource)
+
+    def _create_operand_request_dependencies(
+        self, cpd_operators_project: str, operator_name: str, subscription_metadata: SubscriptionMetadata
+    ):
+        """Creates dependent operand requests for the operator
+        corresponding to the given subscription metadata
+
+        Parameters
+        ----------
+        cpd_operators_project
+            IBM Cloud Pak for Data operators project
+        operator_name
+            name of the operator for which dependent operand requests shall be
+            created
+        subscription_metadata
+            metadata of the subscription to be created
+        created_dependent_subscriptions
+            list of created dependent subscriptions
+        """
+
+        for operand_request_name in subscription_metadata.operand_request_dependencies:
+            kind_metadata = self._kinds[Kind.OperandRequest]
+
+            if (
+                custom_resource := self._openshift_manager.get_namespaced_custom_resource_if_exists(
+                    cpd_operators_project, operand_request_name, kind_metadata
+                )
+            ) is not None:
+                if self._custom_resource_status_value_is_as_expected(custom_resource, "phase", "Running"):
```

Comment:
Running is also a word that we can extract as constant... maybe create some enum with typical statuses?